conf/sample.ini

```diff
@@ -15,16 +15,16 @@
 #################################### Paths ####################################
 [paths]
 # Path to where grafana can store temp files, sessions, and the sqlite3 db (if that is used)
-;data = /var/lib/grafana
+;data = data
 
 # Temporary files in `data` directory older than given duration will be removed
 ;temp_data_lifetime = 24h
 
 # Directory where grafana can store logs
-;logs = /var/log/grafana
+;logs = data/log
 
 # Directory where grafana will automatically scan and look for plugins
-;plugins = /var/lib/grafana/plugins
+;plugins = data/plugins
```

Comment:
?

Reply:
was wrongly checked it, reverting now. this file can be ignored, will disappear in a sec.